packages/plugins/solidity-tests/src/lib/index.js

```diff
@@ -152,7 +155,23 @@ class SolidityTestRunner {
             evm: { methodIdentifiers: instance.functionHashes }
           };
 
-          remixTests.runTest(contract, contractObj, details, {accounts}, reporter.report.bind(reporter), cb);
+          const ast = allCompiledContracts[instance.className].ast;
+
+          // TODO: temporary fix until this is fixed in remix-tests, remove this snippet when it is
```

Comment:
Could a link be included to the relevant `remix-tests` issue/comment/PR on GitHub?